testing/run_api_tests.sh

```diff
@@ -91,5 +122,9 @@ if [[ $MIGRATION -eq 1 ]]; then
     kill -TERM $pid
 fi
 
+if [ "$docker_down" -eq 1 ]; then
```

Comment:
Tests run through, but I'm still getting this error (e.g. when running `bash testing/run_api_tests.sh sqlite`).

```
[...]
Shutting down Wakapi ...
testing/run_api_tests.sh: line 125: [: : integer expression expected
Exiting with status 0
```

Reply:
Should be addressed by default value, if not defined.